lib/git_reflow/workflows/core.rb

```diff
@@ -15,8 +15,9 @@ class Core
       command(:setup, defaults: {local: false, enterprise: false}) do |**params|
         reflow_options             = { project_only: params[:local], enterprise: params[:enterprise] }
         existing_git_include_paths = GitReflow::Config.get('include.path', all: true).split("\n")
+        evaluated_config_path = GitReflow::Config::CONFIG_FILE_PATH.gsub(/\$([a-zA-Z_]+[a-zA-Z0-9_]*)|\$\{(.+)\}/) { ENV[$1 || $2] }
```

Comment:
I think it makes sense to update the constant.  There are a few other places that reference it.